### PR TITLE
Assorted JS tweaks

### PR DESCRIPTION
**Pull request by XhmikosR**
**By submitting this pull request, I confirm the following:** `{please fill any appropriate checkboxes, e.g: [X]}`

`{Please ensure that your pull request is for the 'devel' branch!}`

- [ ] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [ ] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [ ] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [ ] I give this submission freely and claim no ownership.
- [ ] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

**What does this PR aim to accomplish?:**

`{A detailed description, screenshots (if necessary), as well as links to any relevant GitHub issues}`

**How does this PR accomplish the above?:**

`{A detailed description (such as a changelog) and screenshots (if necessary) of the implemented fix}`

**What documentation changes (if any) are needed to support this PR?:**

`{A detailed list of any necessary changes}`

> * `{Please delete this quoted section when opening your pull request}`
> * You must follow the template instructions. Failure to do so will result in your issue being closed.
> * Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.
> * Detail helps us understand an issue quicker, but please ensure it's relevant.

### CODE REVIEW

**Comment by XhmikosR** [comment]
I plan to squash these if everything is fine later. I just couldn't test it myself, so please try as many things as possible and let me know of any Browser console errors.

File: scripts/pi-hole/js/network.js [hunk 1]
```diff
@@ -66,17 +60,17 @@ function mixColors(ratio, rgb1, rgb2)
 
 $(document).ready(function() {
     tableApi = $("#network-entries").DataTable( {
-        "rowCallback": function( row, data, index )
+        "rowCallback": function( row, data )
         {
-             var color, mark, lastQuery = parseInt(data["lastQuery"]);
+             var color, mark, lastQuery = parseInt(data.lastQuery);
```
Comment (DL6ER) [inline scripts/pi-hole/js/network.js]: I see that a lot (maybe even the majority) of changes in this PR is
```diff
- some["a"]
+ some.a
```

Is there any advantage in this change else than using two characters less?

Comment (XhmikosR) [reply scripts/pi-hole/js/network.js]: https://eslint.org/docs/rules/dot-notation

Also, in the past it was more perfromant in some cases.

**Comment by DL6ER** [comment]
Your last commit does not include a Signed-Off line. Also, it would be nice if you could put in some real content into the PR description.

**Comment by XhmikosR** [comment]
The last patch is supposed to be edited later. I haven't tested and I was
waiting for promofaux to test it out. I will revisit this branch but
remember it's untested.

**Comment by PromoFaux** [comment]
@XhmikosR all working so far. I'll hold off the approval for now as it appears you still have more to add?

**Comment by XhmikosR** [comment]
Nope I don't plan to push anything else if this works. But please triple
check everything and if you merge squash the patches  :)

**Comment by XhmikosR** [comment]
@PromoFaux can you try one more time please? I moved my other changes to another branch for later.

**Comment by DL6ER** [comment]
This PR has a merge conflict.

**Comment by XhmikosR** [comment]
I already rebased this. But please before merging it try it one more time.

**Comment by PromoFaux** [comment]
OK, I've been through every page and pressed every button and have not come up against anything that is no longer working... (also nothing to note in js console)

**Pull request merged**